Performance/perf_runner.sh

```diff
@@ -41,22 +41,52 @@ readonly GOOGLE_PROTOBUF_CHECKOUT=${GOOGLE_PROTOBUF_CHECKOUT:-"$script_dir/../..
 
 function usage() {
   cat >&2 <<EOF
-Usage: $0 [-p <true|false>] [-s2|-s3] <field count> <field types...>
-
-Currently supported field types:
-    int32, sint32, uint32, fixed32, sfixed32,
-    int64, sint64, uint64, fixed64, sfixed64,
-    float, double, string,
-    ...and repeated variants of the above.
-
-    Additionally, you can specify "all" to run the harness
-    multiple times with all of the (non-repeated) field types
-    listed above.
-
-Options:
-    -p <true|false>: Adds a packed option to each field.
-    -s[23]:          Generate proto2 or proto3 syntax. proto3 is
-                     the default.
+SYNOPSIS
+    $0 [-c <rev|c++> -c <rev|c++> ...] [-p <true|false>] [-s2|-s3] <field count> <field types...>
+
+    Currently supported field types:
+        int32, sint32, uint32, fixed32, sfixed32,
+        int64, sint64, uint64, fixed64, sfixed64,
+        float, double, string,
+        ...and repeated variants of the above.
+
+        Additionally, you can specify "all" to run the harness multiple
+        times with all of the (non-repeated) field types listed above.
+        ("all" is not compatible with revision comparisons.)
```

Comment:
Especially for revision comparisons, should we look at adding a "mixed" or something that is a message with say two dozen fields of differing types and also uses messages within messages to get some depth?